os_net_config/impl_ifcfg.py

```diff
@@ -306,38 +305,58 @@ def iproute2_apply_commands(self, device_name, filename, data):
-        logger.debug("File values:\n%s" % file_values)
-        logger.debug("Data values:\n%s" % data_values)
+        logger.debug(f'File values:\n{file_values}')
+        logger.debug(f'Data values:\n{data_values}')
         changes = self.enumerate_ifcfg_changes(file_values, data_values)
         commands = []
         new_cidr = 0
         old_cidr = 0
         # Convert dot notation netmask to CIDR length notation
-        if "NETMASK" in file_values:
-            netmask = file_values["NETMASK"]
+        if 'NETMASK' in file_values:
+            netmask = file_values['NETMASK']
             old_cidr = netaddr.IPAddress(netmask).netmask_bits()
-        if "NETMASK" in data_values:
-            netmask = data_values["NETMASK"]
+        if 'NETMASK' in data_values:
+            netmask = data_values['NETMASK']
             new_cidr = netaddr.IPAddress(netmask).netmask_bits()
-        if "IPADDR" in changes:
-            if changes["IPADDR"] == "removed" or changes[
-                "IPADDR"] == "modified":
+        if 'IPADDR' in changes:
+            if changes['IPADDR'] == 'removed' or \
+               changes['IPADDR'] == 'modified':
                 if old_cidr:
-                    commands.append("addr del %s/%s dev %s" %
-                                    (file_values["IPADDR"], old_cidr,
-                                     device_name))
+                    ip = file_values['IPADDR']
+                    commands.append(f'addr del {ip}/{old_cidr} dev {dev_name}')
                 else:
                     # Cannot remove old IP specifically if netmask not known
-                    commands.append("addr flush dev %s" % device_name)
-            if changes["IPADDR"] == "added" or changes["IPADDR"] == "modified":
-                commands.insert(0, "addr add %s/%s dev %s" %
-                                (data_values["IPADDR"], new_cidr, device_name))
-        if "MTU" in changes:
-            if changes["MTU"] == "added" or changes["MTU"] == "modified":
-                commands.append("link set dev %s mtu %s" %
-                                (device_name, data_values["MTU"]))
-            elif changes["MTU"] == "removed":
-                commands.append("link set dev %s mtu 1500" % device_name)
+                    commands.append(f'addr flush dev {dev_name}')
+            if changes['IPADDR'] == 'added' or changes['IPADDR'] == 'modified':
+                ip = data_values['IPADDR']
+                commands.insert(0, f'addr add {ip}/{new_cidr} dev {dev_name}')
+        if 'MTU' in changes:
+            if changes['MTU'] == 'added' or changes['MTU'] == 'modified':
+                mtu = data_values['MTU']
+                commands.append(f'link set dev {dev_name} mtu {mtu}')
+            elif changes['MTU'] == 'removed':
+                commands.append(f'link set dev {dev_name} mtu 1500')
+        if 'DOMAIN' in changes:
+            old = file_values['DOMAIN']
+            new = data_values['DOMAIN']
```

Comment:
Domain should be like https://github.com/os-net-config/os-net-config/blob/master/os_net_config/impl_ifcfg.py#L759